guacamole-ext/src/main/java/org/apache/guacamole/token/StandardTokens.java

```diff
@@ -74,6 +76,11 @@
      */
     private static final String TIME_FORMAT = "HHmmss";
 
+    /**
+     * The prefix of the arbitrary attribute tokens.
+     */
+    public static final String ATTR_TOKEN_PREFIX = "GUAC_ATTR:";
```

Comment:
Okay, one of the things I figured out is that the RegEx for tokens won't currently match on ":", so, either one of two things will need to be done:
- Modify the RegEx in `TokenFilter` and add the match for ":" to that.
- Change this so that it fits into the current RegEx, which would probably mean using "GUAC_ATTR_" and then the tokens become GUAC_ATTR_MAIL, GUAC_ATTR_WORKSTATIONNAME, etc.

Reply:
I think I'll change it so it fits into the current RegEx.